### ISSUE TITLE
Support to WCAG 2.1

### ISSUE THREAD

**Issue by Itakora** [w3c/wcag-em-report-tool#293]
Hi,
is there anybody working on adapting this tool into the new recommendation? Is there a plan to do it?
Regards.

**Comment by Itakora** [comment]
Worked on it, i have sent a pull request with the new criteria. Hopes this helps to accelerate the release.
Regards,
Olga

**Comment by nitedog** [comment]
Hi Olga,

Actually there is already a WCAG 2.1 in JSON format:
 - https://github.com/w3c/wai-wcag-quickref/blob/gh-pages/_data/wcag21.json

The issue is really how we want to address both WCAG 2.0 and WCAG 2.1 at the same time, because they are both operational versions. We need to work on the references in the tool and also some switch in the user interface. This work is scheduled for Q4.

Best,
  Shadi

**Comment by tiffy23** [comment]
Hello,

Is there an estimated date when the adapted WCAG EM Report Tool for WCAG 2.1 will be available?

Thank you in advance!
Best,
Stephanie Jagl-Posch

**Comment by scottsb** [comment]
For whenever this is actually addressed officially: the JSON spec file cited by @nitedog above uses different criterion IDs than the current version of this tool. That means that if it's adopted as-is, reports generated by previous versions of the tool will not load in the new version. Ideally either the existing IDs would be retained, or some compatibility layer would be instituted.

**Comment by jo9ty60** [comment]
Any update on when the WCAG EM Report Tool for WCAG 2.1 will be available to use? I'm unable to use the JSON spec file.

Thanks!

**Comment by rvantonisse** [comment]
People that are interrested in progress to full 2.1 support can follow activity on (draft)Pull request #301 . This branch has implemented the wcag21.json specification file from the [wai-wcag-quickref](https://github.com/w3c/wai-wcag-quickref), as suggested by nitedog, which makes it possible to do 2.1 wcag evaluations.

There are a few gotchas though:
- it cannot load previously created WCAG 2.0 evaluation data (Due to new succescriteria id's under the hood for WCAG 2.1)
- links to WCAG specification are not correct (due to previous gotcha)

When the gotchas are tackled, full WCAG 2.1 support will be there. This is expected to be solved this month (September).

**Comment by rvantonisse** [comment]
Work can be previewed at github pages: https://w3c.github.io/wcag-em-report-tool/dist/#!/

Works with all evaluation data generated by previous versions of the tool.

**Comment by rvantonisse** [comment]
Ready in [release-2.0.0](https://github.com/w3c/wcag-em-report-tool/tree/release-2.0.0)
